lib/utils/frc_teams.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:neatteam_scouting_2023/models/team.dart';
+import 'package:neatteam_scouting_2023/utils/utils.dart';
+
+class FrcTeams {
+  static Future<List<Team>> getAllTeams() async {
+    String teamsFileData = await loadAsset('assets/teams.csv');
+    return teamsFileData.split("\n").map((teamData) {
+      List<String> team = teamData.split(",");
+      return Team(number: int.parse(team[0]), name: team[1]);
+    }).toList();
```

Comment:
I tried doing that but it would be more complex for the specific csv file. The function ```CsvToListConverter().convert(file)``` function converts the file to ```List<List<dynamic>>``` and after that I'll need to take the "first" (and only) list and convert the contents of it to the ```Team``` object. Correct me if I'm wrong but it's easier to simply split it to lines and then convert every line to a ```Team``` object.